docs/user-guide/global-configurations/authorization/user-access.md

```diff
@@ -188,6 +188,10 @@ Click `Save`once you have configured all the required permissions for the users.
 
 ### Edit User Permissions
 
+{% hint style="warning" %}
+You cannot edit user permissions if 'auto-assign permission' is enabled in the SSO configuration of either [Microsoft](../authorization/sso/microsoft.md#auto-assign-permissions) or [LDAP](../authorization/sso/ldap.md#auto-assign-permissions).
```

Comment:
Rephrase suggestion: Direct user permissions cannot be edited if you're using LDAP/Microsoft for SSO and 'auto-assign permission' is enabled. Permissions can only be managed via permission groups in such a scenario. //Hyperlink to relevant section in permission groups?